src/components/FeesField/BitcoinKind.js

```diff
@@ -147,7 +147,13 @@ class FeesField extends Component<OwnProps, State> {
           onChange={onChange}
           onChangeFocus={this.onChangeFocus}
           loading={!feePerByte && !error}
-          error={!feePerByte && error ? new FeeNotLoaded() : null}
+          error={
+            !feePerByte && error
+              ? new FeeNotLoaded()
+              : feePerByte && feePerByte.isZero()
+                ? new Error(t('errors.FeesAreRequired.title'))
```

Comment:
I would prefer we define a new `@ledgerhq/errors` for this. `FeesRequired` , can you provide a PR in ledgerjs? i can merge it quick.

Reply:
yes sir